# typed: true
def fun(arg); end;
def foo; end;
next fun foo do end # error: No `do` block around `next`
